zhaquirks/tuya/builder/__init__.py

```diff
@@ -93,13 +93,15 @@ def tuya_battery(
     def tuya_metering(
         self,
         dp_id: int,
+        converter: Optional[Callable[[Any], Any]] = None,
         metering_cfg: TuyaLocalCluster = TuyaValveWaterConsumed,
     ) -> QuirkBuilder:
         """Add a Tuya Metering Configuration."""
         self.tuya_dp(
             dp_id,
             metering_cfg.ep_attribute,
-            "current_summ_delivered",
```

Comment:
I'd prefer to match the other methods and use scale here. If we need a complex conversion, then follow the `tuya_dp` then `adds` flow.